keyboards/suihankey/split/rev1/rev1.h

```diff
@@ -25,7 +25,7 @@
  * The second converts the arguments into a two-dimensional array which
  * represents the switch matrix.
  */
-#define LAYOUT( \
+#define LAYOUT_split_3x5_3( \
```

Comment:
This should have a `#define LAYOUT LAYOUT_split_3x5_3` alias at the bottom of the file for out-of-tree keymaps.

Reply:
With out-of-tree keymaps you mean keymaps that haven't yet been merged into master, right? Why would the responsibility of making those keymaps compatible with master (after this pull request is merged) fall on me instead of those that made those keymaps? Before a pull request gets merged, I think it should be the pull request author's responsibility to merge master into their branch and make sure everything works and is compatible. Is that not the way it goes with QMK?

Reply:
Because not everyone is proficient with git, or wants to add their keymap to the repo.

So out of respect, adding backwards compatibility by adding a define is a simple way to ensure that nothing breaks.

The alternative is retargeting to develop and label it as a breaking change, which means longer for the changes to hit master.

Reply:
That actually does make sense, and I hadn't thought about that. Thanks for the clarification. I'll put the alias in right now. Hopefully that'll be the last thing for this pull request now.